ibm/service/atracker/resource_ibm_atracker_target_test.go

```diff
@@ -62,10 +79,21 @@ func testAccCheckIBMAtrackerTargetConfigBasic(name string, targetType string) st
 				target_crn = "crn:v1:bluemix:public:cloud-object-storage:global:a/11111111111111111111111111111111:22222222-2222-2222-2222-222222222222::"
 				bucket = "my-atracker-bucket"
 				api_key = "xxxxxxxxxxxxxx"
-				service_to_service_enabled = false
+				service_to_service_enabled = true
 			}
+			logdna_endpoint {
```

Comment:
For a single Atracker instance  can you add more than one endpoint?

Reply:
No, it can have only one endpoint